cubed/array_api/array_object.py

```diff
@@ -31,6 +37,55 @@ def __array__(self, dtype=None) -> np.ndarray:
     def __repr__(self):
         return f"cubed.Array<{self.name}, shape={self.shape}, dtype={self.dtype}, chunks={self.chunks}>"
 
+    def _repr_html_(self):
+        try:
+            grid = self.to_svg(size=ARRAY_SVG_SIZE)
+        except NotImplementedError:
+            grid = ""
+
+        if not math.isnan(self.nbytes):
+            nbytes = format_bytes(self.nbytes)
+            cbytes = format_bytes(math.prod(self.chunksize) * self.dtype.itemsize)
```

Comment:
(I thought I had made this comment in the review I did yesterday, but I can't find it now. Anyway, it's fine if we address it as a follow up if you want to merge this now.)

We should probably use Cubed's [`memory_repr`](https://github.com/tomwhite/cubed/blob/0f2fe5afff27789c78d80f48e6224dd30eed7779/cubed/utils.py#L51-L70) for consistency with the DAG visualization.

Reply:
Thanks for fixing!